crates/api_models/src/subscription.rs

```diff
@@ -174,10 +192,16 @@ pub struct ConfirmSubscriptionRequest {
     pub customer_id: common_utils::id_type::CustomerId,
 
     /// Billing address for the subscription.
-    pub billing_address: Option<Address>,
+    pub billing: Option<Address>,
+
+    /// Shipping address for the subscription.
+    pub shipping: Option<Address>,
 
     /// Payment details for the invoice.
     pub payment_details: PaymentDetails,
+
+    /// Merchant specific Unique identifier.
+    pub merchant_reference_id: Option<String>,
```

Comment:
These information already available in the backend, need not to be sent from the client